cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2401,6 +2401,14 @@
     </description>
   </property>
 
+  <property>
+    <name>router.audit.path.check.enabled</name>
```

Comment:
settings in cdap-default.xml need to be in alphabetical order, and you need to update DEFAULT_XML_MD5_HASH in cdap-docs/admin-manual/build.sh with the new md5 of cdap-default.xml